internal/rbd/driver/driver.go

```diff
@@ -159,6 +159,8 @@ func (r *Driver) Run(conf *util.Config) {
 		r.cs = NewControllerServer(r.cd)
 		r.cs.ClusterName = conf.ClusterName
 		r.cs.SetMetadata = conf.SetMetadata
+		log.WarningLogMsg("replication service running on controller server is deprecated " +
```

Comment:
can you create an issue for this, and add the link in the message? users that read the message can then get additional details easily. Something like `the internal Replication Service is deprecated and replaced by CSI-Addons, see https://github.com/ceph/ceph-csi/12345 for more details`.

Reply:
done